### PR TITLE
feat: add community token list

### LINKED ISSUE #996: Create a "Community List"
Consider a community list so new tokens with strong community support can be added to the UI without cluttering the original balanced UI.

### PR DESCRIPTION
**Pull request by hetfly**
close #996

### CODE REVIEW

**Comment by parrot9design** [comment]
To remove the community token list:

- Replace the `Add community token list` text link with `Remove community token list`.
- Use this for the confirmation modal:
Screen Shot 2022-08-30 at 11 32 25 AM

```
<div class="modal">
	<div class="modal-content">
		<!-- Overview -->
		<h3 class="label text-center">Remove community token list?</h3>
		<p class="text-center">You'll only be able to interact with the default assets listed on Balanced.</p>
		<div class="modal-actions">
			<a href="#" class="cancel modal-close">Cancel</a> <a href="./" class="button">Remove token list</a>
		</div>
	</div>
</div>
```

**Comment by parrot9design** [comment]
As for people who've already supplied liquidity for tokens that will be moved to the community token list, we could show a banner message directly above the liquidity details/liquidity pools section, rather than at the top of the page where it's more likely to be ignored:

Screen Shot 2022-08-30 at 11 45 48 AM

```
<div class="banner" style="margin-bottom: 30px;">
	<div class="grid grid-center">
		<div class="grid-cell">
			<p class="label">Missing a liquidity pool? <a href="#">Add the community token list</a> to see details for all pools you've participated in.</p>
		</div>
		<div class="grid-cell text-right close">
			<a href="#">
				<img src="/img/icon/cross.svg">
			</a>
		</div>
	</div>
</div>
```

**Comment by parrot9design** [comment]
Let's add CLAW and VOID to this list, as well. See https://github.com/balancednetwork/balanced-network-interface/pull/1071 for the CLAW token details.

Here are the details for VOID:

Token name: Void Token
Token symbol: VOID
Mainnet contract: cx32d5b6c3eff488b058feff88acec824abe7ed8ca 
Testnet contract: cxc2faf30ef79327f8a177f4b236ee3a1db8de81ec

Token icon:
void

**Comment by parrot9design** [comment]
Is it possible to add a version of these assets for the testnet, as well? For VOID, this would be `cxc2faf30ef79327f8a177f4b236ee3a1db8de81ec1`, unsure about the others.

---
There's a `the` missing from the banner text. Should be `Add the community token list`:
Screen Shot 2022-08-31 at 5 04 14 PM

---

When I don't have the community token list, I see the banner message and the liquidity details section:
Screen Shot 2022-08-31 at 5 05 58 PM

After I add it, the liquidity details section disappears:
Screen Shot 2022-08-31 at 5 06 09 PM

Might be related to #1074?

**Comment by hetfly** [comment]
@parrot9design updated

Re the testnet tokens - no problem, just need the addresses
Screen Recording 2022-08-31 at 14.25.06

**Review by parrot9design — APPROVED** [review]

**Comment by hetfly** [comment]
@shallcdn or @RiseBlock please review

File: src/app/index.tsx [hunk 1]
```diff
@@ -38,9 +35,9 @@ export function App() {
         <NotificationContainer />
         <WalletModal />
         {/* Add message for community */}
-        <Banner messageID={'continuousFees'} location={PageLocation.HOME}>
+        {/* <Banner messageID={'continuousFees'} location={PageLocation.HOME}>
           <Message />
-        </Banner>
+        </Banner> */}
```
Comment (shallcdn) [inline src/app/index.tsx:40]: It's should be removed

**Review by shallcdn — APPROVED** [review]
LGTM

File: src/app/index.tsx (hunk 1 above)
Comment (hetfly) [reply src/app/index.tsx]: done

**Pull request merged**